packages/ui-app/src/AddressInfo.tsx

```diff
@@ -103,6 +105,19 @@ class AddressInfo extends React.PureComponent<Props> {
     );
   }
 
+  private renderBonded (bonded: true | Array<BN>, stakingLedger: StakingLedger) {
```

Comment:
I'm intrigued by this function. In which use case would we receive an `Array<BN>`? The next `if` also made me ask "what does `bonded` really mean here, if it's true"

It'd be great to have a couple comments inline